### DISCUSSION TITLE
1.2 Release

### DISCUSSION THREAD

**Discussion by s0undt3ch** [python-poetry/poetry#3950] [General] [answered]
When can we expect a 1.2 release?

Mostly interested in the plugins system which should be released on 1.2

**Comment by abn** [comment]
Unfortunately we do not have a hard timeline yet for the 1.2.0 release. However, you can expect the first alpha release, likely within this month. Going by the `1.1.0` release, we might have 2-3 alpha and 1-2 beta releases followed by a release candidate prior to `1.2.0`.

In the meantime, you are welcome use the master branch for plugin development.

**Reply by s0undt3ch** [reply]
Unfortunately my perspective is not that of a plugin developer, it's an end user perspective which needs versions from SCM tags.
I'll wait.

**Reply by abn** [reply]
The plugin feature is in its early days. We hope to get some feedback once the first alpha is cut. We do anticipate issues to crop up along the way. Hopefully minor ones. 

We definitely would appreciate more end users using that version and reporting issues.

**Reply by k-sap** [reply]
Hi, I'd like to ask if there was some progress with poetry v 1.2.  There was no further releases.

**Comment by k-sap** [comment]
Hi, I'd like to ask if there was some progress with poetry v 1.2. There was no further releases.

**Comment by marianhlavac** [comment]
Still no news on when we can expect the v1.2 release? The features sounds promising and we can't wait!

**Comment by jdinunzio** [comment]
Any news about this? Having a pip installable poetry 1.2 would be very useful.

**Reply by edgarrmondragon** [reply]
The betas are pip-installable 🙂 

```
pip install 'poetry==1.2.0b3'
```

**Comment by ghost** [comment]
How close is Beta 3 to the final release version?

**Comment by neersighted** [comment, marked as answer]
The canonical location for tracking the status of the 1.2 release is https://github.com/python-poetry/poetry/issues/5586 -- please keep the discussion there minimal as to avoid generating noise for subscribers.
